advanced/events.md

```diff
@@ -26,14 +26,14 @@ tinymce.init({
 });
 ```
 
-The following example uses a function to create a console log entry when text color is changed in the editor. This is also an example of handling an event that returns data.
+The following example uses a function to create a console log entry a command is executed in the editor. This is also an example of handling an event that returns data.
```

Comment:
Should this be `console log entry when a command...`?

Reply:
:facepalm: I shouldn't make edits after 6pm. Thanks!

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1658/commits/4f0273d9c520a9eaa00a46d1ff4a9a69fa8f31aa